Main.tex

```diff
@@ -15,50 +15,19 @@
- in wichtiger Aspekt, der bei der Verwendung von bildgenerierenden KI-Systemen berücksichtigt werden muss, ist die Möglichkeit, 
- dass ethisch fragwürdige Inhalte generiert werden können. Es gibt ein wachsendes Bewusstsein für die Notwendigkeit, 
- sicherzustellen, dass KI-Systeme keine rassistischen, sexistischen oder anderweitig diskriminierenden Inhalte generieren. Diese 
- Herausforderung stellt eine wichtige Forschungsfrage dar, die untersucht werden muss, um sicherzustellen, dass die Verwendung von
-  bildgenerierenden KI-Systemen ethisch vertretbar bleibt.
-
- \section{Motivation und Relevanz}
- Die Entwicklung von AI-Kunst reicht zurück bis in die 1960er Jahre\cite{Garcia}, aber erst in den letzten Jahren wurden 
- diese KIs durch die Veröffentlichungen von bahnbrechenden Projekten wie DALL-E (01/21), Midjourney (03/22) oder Stable Diffusion 
- (08/22) der Allgemeinheit zugänglich gemacht. Auslotungen der Leistungsfähigkeit dieser KIs zeigen jedoch auch auf, 
- dass sie sexistische oder rassistische Verzerrungen aufweisen \cite{Schmidt}und für moralisch fragwürdige Zwecke missbraucht werden 
- können, z. B. bei der Erstellung von Deepfakes, pornografischen oder gewalttätigen Inhalten.\cite{Hadero}
-
- Angesichts dessen haben u. a. die Entwickler von Midjourney reagiert und das Verwenden von Wörtern mit Bezug zur 
- Thematik menschlicher Fortpflanzungssysteme verboten \cite{Heikkilae} und ihre kostenlose Testversion eingestellt. 
- \cite{NelsonMidjourney} Doch es stellt sich die 
- Frage, wie Anbieter und Entwickler solcher KIs gegen diese schädliche Verwendung ihrer Technologie vorgehen können.
- 
- Die vorliegende Arbeit wird sich mit dieser Frage auseinandersetzen und untersuchen, welche Möglichkeiten es gibt, um KIs vor 
- Missbrauch zu schützen. Sie wird diskutieren, wie die Anbieter und Entwickler dieser Technologie verantwortungsbewusst 
- handeln können, um sicherzustellen, dass ihre KIs nur für ethisch vertretbare Zwecke eingesetzt werden. Das Ziel ist es, 
- Wege aufzuzeigen, wie KI-Entwickler dazu beitragen können, die positiven Auswirkungen der Technologie zu maximieren, 
- während sie gleichzeitig die negativen Auswirkungen minimieren.
- \section{Forschungsfrage}
- In dieser Arbeit soll die Frage geklärt werden welche Methodiken möglich sind bzw. angewandt werden, um die Erzeugung ethisch 
- fragwürdiger Inhalte durch IGAI zu verhindern?
+
+Die Entwicklung von Kunst mithilfe von KI reicht zurück bis in die 1960er Jahre\cite{Garcia}, aber erst in den letzten Jahren wurden diese KIs durch die Veröffentlichungen von Projekten wie DALL-E (01/21), Midjourney (03/22) oder Stable Diffusion (08/22) der Allgemeinheit zugänglich gemacht. Auslotungen der Leistungsfähigkeit dieser KIs zeigen jedoch auf, dass sie sexistische oder rassistische Verzerrungen aufweisen \cite{Schmidt}und für moralisch fragwürdige Zwecke missbraucht werden können, z. B. zur Erstellung von Deepfakes, pornografischen oder gewalttätigen Inhalten.\cite{Hadero}
```

Comment:
"in den letzten Jahren" hatte Herrn Müller ja nicht geschmeckt, ist das ok dadurch, dass die Monate und Jahre bei den Sachen dahinter stehen, oder lieber "in den letzten zwei Jahren"?

Reply:
würde einfach den zeitraum von chatgpt veröffentlichung und heute nehmen oder so. Auf jeden Fall einen konkreten Zeitraum